node.go

```diff
@@ -142,37 +162,43 @@ func (n *Node) Restart() error {
 	n.snapshotter = snap.New(n.snapdir)
 	snapshot, err := n.snapshotter.Load()
 	if err != nil && err != snap.ErrNoSnapshot {
-		return fmt.Errorf("casbin: error loading snapshot (%v)", err)
+		return fmt.Errorf("casbin: Failed loading snapshot (%v)", err)
```

Comment:
I recommend use `https://github.com/pkg/errors`.

The code like `errors.Wrap(err, "Failed loading snapshot")`.

Reply:
> I recommend use `https://github.com/pkg/errors`.
> 
> The code like `errors.Wrap(err, "Failed loading snapshot")`.

it likes good, I will try